contracts/plugins/assets/frax-eth/SFraxEthCollateral.sol

```diff
@@ -23,17 +24,27 @@ import "./vendor/IsfrxEth.sol";
  */
 contract SFraxEthCollateral is AppreciatingFiatCollateral {
     using OracleLib for AggregatorV3Interface;
+    using FraxOracleLib for FraxAggregatorV3Interface;
```

Comment:
Should we remove the comment that "THIS PLUGIN IS NOT READY TO BE USED IN PRODUCTION"?